bin/run_docker.py

```diff
@@ -117,26 +118,42 @@ def create_log_file(
     If no text is given, it writes "No Logs" to the file.
 
     Arguments:
-        log_filename: The name of the log file to create
-        log_text: The text to write to the log file. If given as a byte string,
-                  it will be decoded as UTF-8 before being written.
+        log_file_name: The name of the log file to create
+        log_max_size: The maximum size of the log file in kilobytes
+        log_file_path: The path where the log file will be created.
+                       If not specified, the current working directory will be used.
+        log_text: The text to write to the log file
 
     """
-    if not log_file_path:
+    if log_file_path is None:
         log_file_path = os.getcwd()
 
+    if log_text is None:
+        log_text = "No Logs"
+    elif isinstance(log_text, bytes):
+        log_text = log_text.decode("utf-8")
+
+    # We need the size of the log message in bytes
+    log_text_size = len(log_text.encode("utf-8"))
```

Comment:
minor enhancement - If the text is already in bytes you can do the `len` check then. There is no need to decode, then encode it right after.

Reply:
[Updated](https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/pull/20/commits/b908565785ca9ff7ddb1879a49f062e8d54f46e1) and ran [a test](https://tower-dev.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/example-dev-project/watch/2LSoQvPpTEPjl0) to make sure it works. Thanks!